website/docs/docs/deploy/ci-jobs.md

```diff
@@ -38,8 +37,13 @@ To make CI job creation easier, many options on the **CI job** page are set to d
     - **Commands** &mdash; By default, it includes the `dbt build --select state:modified+` command. This informs dbt Cloud to build only new or changed models and their downstream dependents. Importantly, state comparison can only happen when there is a deferred environment selected to compare state to. Click **Add command** to add more [commands](/docs/deploy/job-commands)  that you want to be invoked when this job runs.
     - **Linting**<Lifecycle status="beta" /> &mdash; Enable this option for dbt to [lint the SQL files](/docs/deploy/continuous-integration#sql-linting) in your project as the first step in `dbt run`. If this check runs into an error, dbt can either **Fail job run** or **Continue running job**. 
     - **Run compare changes**<Lifecycle status="enterprise" /> &mdash; Enable this option to compare the last applied state of the production environment (if one exists) with the latest changes from the pull request, and identify what those differences are. To enable record-level comparison and primary key analysis, you must add a [primary key constraint](/reference/resource-properties/constraints) or [uniqueness test](/reference/resource-properties/data-tests#unique). Otherwise, you'll receive a "Primary key missing" error message in dbt Cloud.
-    
+
       To review the comparison report, navigate to the [Compare tab](/docs/deploy/run-visibility#compare-tab) in the job run's details. A summary of the report is also available from the pull request in your Git provider (see the [CI report example](#example-ci-report)). 
+
+      :::info Optimization tip 
+      When you enable the **Run compare changes** checkbox, you can customize the comparison command to optimize your CI job. For example, if you have large models that take a long time to compare, you can exclude them to speed up the process using the `--exclude` flag. Use [custom commands](/docs/deploy/job-commands#advanced-ci-compare-changes) to modify the selection criteria.
+      :::
```

Comment:
I wonder if event_time belongs in this same callout box. That way, we could give tips for both model selection and event time, and both can link out to separate pages. 

@joellabes should be able to help with event time.

Reply:
i thought the user can't do anything with `event_time` yet though bc the `--sample` flag (name tbd) won't be available until feb 2025?

Reply:
Today, event_time is used if set to do an "apples with apples" comparison. If you've hand-rolled a sampling thing in CI (e.g. [with this](https://docs.getdbt.com/best-practices/best-practice-workflows#limit-the-data-processed-when-in-development)) then setting event_time will let advanced CI filter to only the overlapping date ranges from the two tables